mitra_gcm.sage

```diff
@@ -134,8 +137,8 @@ ciphertext = b''.join(ct_blocks)
 print(f'Key1: {hexlify(key1)}')
 print(f'Key2: {hexlify(key2)}')
 print(f'Nonce: {hexlify(nonce)}')
-print(f'AdditionalData: {hexlify(additional_data)}')
```

Comment:
I think we should keep AdditionalData here, as that's also how it's called in the CLI argument.

Reply:
done